types/weekdayname.pp

```diff
@@ -1,3 +1,3 @@
 # Short names for each day of the week.
 type Cron::Weekdayname = Enum['Sun','Mon','Tue','Wed',
-                              'Thu','Fri','Sat']
+'Thu','Fri','Sat']
```

Comment:
if we're changing this already, it would probably be better readable either on one line, or else one day per line